python/feathub/processors/flink/table_builder/redis_utils.py

```diff
@@ -0,0 +1,72 @@
+#  Copyright 2022 The Feathub Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import glob
+import os
+from typing import Sequence
+
+from feathub.online_stores.redis_online_store import RedisOnlineStore
+from pyflink.table import (
+    TableResult,
+    StreamTableEnvironment,
+    Table as NativeFlinkTable,
+    TableDescriptor as NativeFlinkTableDescriptor,
+)
+
+from feathub.feature_tables.sinks.online_store_sink import OnlineStoreSink
+from feathub.processors.flink.flink_jar_utils import find_jar_lib, add_jar_to_t_env
+from feathub.processors.flink.table_builder.source_sink_utils_common import (
+    get_schema_from_table,
+)
+
+
+def insert_into_redis_sink(
+    store: RedisOnlineStore,
+    t_env: StreamTableEnvironment,
+    table: NativeFlinkTable,
+    sink: OnlineStoreSink,
+    keys: Sequence[str],
+) -> TableResult:
+    add_jar_to_t_env(t_env, *_get_redis_connector_jars())
+
+    redis_sink_descriptor_builder = (
+        NativeFlinkTableDescriptor.for_connector("redis")
+        .schema(get_schema_from_table(table))
+        .option("host", store.props["online_store.redis.host"])
+        .option("port", store.props["online_store.redis.port"])
+        .option("tableName", sink.table_name)
+        .option("keyFields", ";".join(keys))
```

Comment:
Should we also support other options accepted by the Java connector, such as password?